### PR TITLE
feat(commemorative-coin): add content type demo - INNO-2004

### PR DESCRIPTION
**Pull request by kalinchernev**
# PR description

Nothing special to note.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

File: src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+      <Fragment>
+        <SiteHeaderHarmonised
+          {...siteHeader}
+          data-ecl-auto-init="SiteHeaderHarmonised"
+          className="ecl-site-header-harmonised--group2"
+        />
+        <PageHeaderHarmonised {...pageHeader} />
+      </Fragment>
+    )}
+    <div className="ecl-u-pv-3xl">
```
Comment (papegaill) [inline src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx]: should be `ecl-u-pv-xl ecl-u-pv-lg-3xl`

File: src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+      <div className="ecl-container">
+        <div className="ecl-row">
+          <aside className="ecl-col-12 ecl-col-lg-3">
+            <h2 className="ecl-u-type-heading-2 ecl-u-type-color-black ecl-u-d-lg-none ecl-u-mv-none">
+              Coins (324)
+            </h2>
+            <h3 className="ecl-u-type-heading-3 ecl-u-type-color-black ecl-u-mt-l ecl-u-mt-lg-none">
```
Comment (papegaill) [inline src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx:70]: All the filters block is not clear for me, according to mocks, this block should be hidden for mobile and have the possibility of expanding it, via buttons "_Filter by_" / "_Hide the filters_" is this something we need to implement? There is also no submit button on the mocks..which I also find strange.

File: src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+              Filter by
+            </h3>
+            <form>
+              <TextInput id="facet-keywords" label="Keywords" width="m" />
+              <Select
+                id="facet-countries"
+                groupClassName="ecl-u-mt-m"
```
Comment (papegaill) [inline src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx]: should be `ecl-u-mt-l` on all filters input

**Review by papegaill — CHANGES_REQUESTED** [review]

File: src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx (hunk 2 above)
Comment (kalinchernev) [reply src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx:70]: Filters are following the pattern of the Search: https://ec.europa.eu/component-library/ec/content-types/search/usage/

Behaviors and visibility are "inheriting" this.

File: src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx (hunk 3 above)
Comment (kalinchernev) [reply src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx]: Updated

File: src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx (hunk 1 above)
Comment (kalinchernev) [reply src/systems/ec/implementations/react/templates/pages/commemorative-coin/src/CommemorativeCoinPage.jsx]: Updated

**Review by papegaill — APPROVED** [review]

**Pull request merged**